src/pca_gophish.yml

```
@@ -52,12 +52,18 @@
             name: "{{ gophish_group }}"
             gid: 421
             system: yes
-        - name: Change Gophish published port from 3380 to 80
+        - name: Change Gophish published port from 3380 to 443
           ansible.builtin.lineinfile:
             path: "{{ pca_gophish_composition_path }}/docker-compose.yml"
             regexp: "published: 3380$"
-            line: "        published: 80"
+            line: "        published: 443"
             state: present
+        - name: Configure Gophish to use TLS
+          ansible.builtin.replace:
+            path: >-
+              {{ pca_gophish_composition_path }}/secrets/gophish/config.json
+            regexp: '^(\s*"use_tls":\s*)false(\s*)$'
+            replace: '\1true'
```

Based on the regexp do you need to add the other matching group?

```suggestion
            replace: '\1true\2'
```

---

I don't think so, since it's just pointless whitespace.  Second matching group removed in commit 6a23b30.